scripts/analytics/src/modules/cli/libs/helpers/format-date.helper.ts

```diff
@@ -0,0 +1,15 @@
+function formatDate(date: Date): string {
```

Comment:
we already have formatDate helper in shared, we need to use it and we need to pass needed format instead of writing all this logic manually